pkg/services/serviceaccounts/models.go

```diff
@@ -158,6 +158,8 @@ type ServiceAccountProfileDTO struct {
 	Teams []string `json:"teams" xorm:"-"`
 	// example: false
 	IsExternal bool `json:"isExternal,omitempty" xorm:"-"`
+	// example: grafana-app
+	RequiredBy string `json:"requiredBy,omitempty" xorm:"-"`
```

Comment:
Can I challenge this? I would not add it now as we only need this for external service accounts for which we can just extract the plugin name on the frontend. (less code change & no modification of the API)
If we start having more managed service accounts, a RequiredBy field might make more sense, but for now IMHO it's a bit overkill.
Wdyt?

Reply:
I thought about it for a minute. Upon implementation, I realised the prefix doesn't exist in the front end and maybe it shouldn't exist in the front end.

Parsing the prefix should come from the back end and will be scalable in case we decide to have multiple types of prefixes.

Reply:
To me it feels like we're prematurely optimising for a scenario where more types of managed service accounts exist. I think this makes the code slightly more complex than it has to be at this point in time. I'm fine either way though, but would probably prefer to keep it simple and resolve the plugin name in the frontend for now.